.github/workflows/dev-deploy.yml

```diff
@@ -0,0 +1,174 @@
+name: Development Stage
+
+on:
+  workflow_run:
+    workflows: [Unit Test]
+    types: [completed]
+
+env:
+  SAM_TEMPLATE: template.yaml
+  TESTING_STACK_NAME: pdf-encrypt-dev
+  TESTING_REGION: ap-northeast-1
+  TESTING_PIPELINE_EXECUTION_ROLE: arn:aws:iam::${{ secrets.AWS_ACCOUNT_ID }}:role/${{ secrets.TESTING_PIPELINE_EXECUTION_ROLE_NAME }}
+  TESTING_CLOUDFORMATION_EXECUTION_ROLE: arn:aws:iam::${{ secrets.AWS_ACCOUNT_ID }}:role/${{ secrets.TESTING_CLOUDFORMATION_EXECUTION_ROLE_NAME }}
+
+permissions:
+  id-token: write
+  contents: read
+jobs:
+
```

Comment:
@javanlth is this formatting error? Doesn't seem like yaml will accept this as valid formatting

Reply:
Nope this is fine.